Mathlib/Analysis/ODE/Gronwall.lean

```diff
@@ -89,6 +89,19 @@ theorem gronwallBound_continuous_ε (δ K x : ℝ) : Continuous fun ε => gronwa
   · simp only [gronwallBound_of_K_ne_0 hK]
     exact continuous_const.add ((continuous_id.mul continuous_const).mul continuous_const)
 
+/-- The Gronwall bound is monotone with respect to the time variable `x`.
+
+Given `0 ≤ δ`, `0 ≤ ε`, and `0 ≤ K`, if `x₁ ≤ x₂`,
+then `gronwallBound δ K ε x₁ ≤ gronwallBound δ K ε x₂`. -/
+lemma gronwallBound_mono (δ K ε : ℝ) (hδ : 0 ≤ δ) (hε : 0 ≤ ε) (hK : 0 ≤ K) :
```

Comment:
Since delta, K and epsilon are mentioned in hypotheses next, I would make them implicit.